CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Added support for feature flags in opensearch.yml ([#4959](https://github.com/opensearch-project/OpenSearch/pull/4959))
 - Add query for initialized extensions ([#5658](https://github.com/opensearch-project/OpenSearch/pull/5658))
 - Revert 'Added jackson dependency to server' and change extension reading ([#5768](https://github.com/opensearch-project/OpenSearch/pull/5768))
+- Experimental support for extended backward compatiblity in searchable snapshots ([#5812](https://github.com/opensearch-project/OpenSearch/pull/5812))
```

Comment:
Since this feature [was released in 2.5.0](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-2.5.0.md?plain=1#L12) we no longer need to include this in the changelog.